oauth2/handler.go

```diff
@@ -642,6 +642,11 @@ func (h *Handler) TokenHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	h.r.Logger().
+		WithField("id_token", accessResponse.GetExtra("id_token")).
```

Comment:
This would leak confidential data, including PII to log systems and potentially log archives, making it incredibly difficult to comply with GDPR rules around PII deletion. I would thus prefer not to save this.

Reply:
Removed